tests/api/test_product.py

```diff
@@ -3930,28 +3872,28 @@ def test_bulk_unpublish_products(
     assert not any(product.is_published for product in product_list)
 
 
-def test_product_base_price_permission(
-    staff_api_client, permission_manage_products, product
-):
-    query = """
-    query basePrice($productID: ID!) {
-        product(id: $productID) {
-            basePrice {
-                amount
-            }
-        }
-    }
-    """
-    product_id = graphene.Node.to_global_id("Product", product.id)
-
-    variables = {"productID": product_id}
-    permissions = [permission_manage_products]
-
-    response = staff_api_client.post_graphql(query, variables, permissions)
-    content = get_graphql_content(response)
-
-    assert "basePrice" in content["data"]["product"]
-    assert content["data"]["product"]["basePrice"]["amount"] == product.price.amount
+# def test_product_base_price_permission(
```

Comment:
If you want temporary turn off tests you should use skip instead of comment out. 
`@pytest.mark.skip(reason="<Reason>")`